pkg/services/resourcesFilterService.go

```diff
@@ -11,45 +11,18 @@ import (
 
 type RootObject struct {
 	Metadata Metadata `json:"metadata"`
+	Kind     string   `json:"kind"`
```

Comment:
is it added for the sake of this PR or for future features?

Reply:
good catch, I thought I needed it, but at the end I did not